gui/default/syncthing/core/syncthingController.js

```diff
@@ -629,6 +631,14 @@ angular.module('syncthing.core')
             }).error($scope.emitHTTPError);
         }, 2500);
 
+        var refreshLastLocalChanges = debounce(function () {
+            $http.get(urlbase + "/events/disk?limit=20").success(function (data) {
+                $scope.globalChangeEvents = data;
```

Comment:
You might want to store them (or display) in reverse order, as then newest change will be on top.